backends-common/redis/src/test/java/org/apache/james/backends/redis/RedisSentinelExtension.java

```diff
@@ -109,7 +109,7 @@ private String createRedisSentinelURI() {
             sb.append("redis-sentinel://123@");
             sb.append(this.stream().map(container -> container.getHost() + ":" + container.getMappedPort(SENTINEL_PORT))
                 .collect(Collectors.joining(",")));
-            sb.append("?sentinelMasterId=mymaster");
+            sb.append("?sentinelMasterId=mymaster&timeout=3s");
```

Comment:
This requires the extra configuration change (not everyone is aware of it and applies it everywhere)

I would be defensive in keeping the Reactor timeout as the safeguard.

WDYT?

Reply:
I'm no problem with both way
feel free to revert it

Reply:
Change or not change? Status on this?

Reply:
I think we can keep removing reactor timeout.

As we have healthcheck timeout safeguard already https://github.com/apache/james-project/blob/11646b05bbc539492fd3715230c22388174a30ad/server/container/guice/common/src/main/java/org/apache/james/PeriodicalHealthChecks.java#L59.